tests/test_legacy_dlx.c

```diff
@@ -10,7 +10,7 @@
 #define HSIZE 324
 
 
-void read_csv(char filename[], int rows, int cols, int array[rows*cols]) {
+void read_csv(char filename[], int rows, int cols, char array[rows*cols]) {
```

Comment:
l19: the cast to int should be changed to cast to char?

l23: int -> char, avoiding an implicit cast?

Reply:
Changes made!